cvat-ui/src/components/annotation-page/annotation-page.tsx

```diff
@@ -80,17 +80,17 @@ export default function AnnotationPageComponent(props: Props): JSX.Element {
                 <AnnotationTopBarContainer />
             </Layout.Header>
             { workspace === Workspace.STANDARD && (
-                <Layout.Content>
```

Comment:
Why not add style to scss file, for example for class selector `ant-layout-content`?